src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,24 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
-
+    public static final String EXAMPLE = "123, some street, #01-01, 123456";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses should be in the format of"
+                                                                + " Block, Street, Unit, Postal Code";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+?(?=,),.+?(?=,),.+?(?=,),.+";
+    public static final String ADDRESS_SPLIT_REGEX = ",";
+    public static final String SEPARATOR = ", ";
+    public static final int BLOCK_INDEX = 0;
+    public static final int STREET_INDEX = 1;
+    public static final int UNIT_INDEX = 2;
+    public static final int POSTALCODE_INDEX = 3;
```

Comment:
Are these constants necessary? If the purpose is to quickly change `Address` format then perhaps the way `value` is constructed also needs some tweaking?